src/lib/log.c

```diff
@@ -31,15 +31,57 @@ void log_init(void)
 }
 
 
+/* Remove tty escapes for colors from buf.  */
+static void remove_escapes()
```

Comment:
The lack of any bounds checking in this function is a bit of a concern. If this is safe by construction due to other places in the code then at least a comment to explain how/why is needed. But even better would be to arrange that this function is locally safe and not capable of running off the end of the array.

Reply:
My `Isn't buf the guest console log?` further down applies to the massaging done here too I think.

Reply:
I'm not sure I follow here: the treatment only strips contents, so it cannot go wrong.  Do you want to me to add an `assert(dst <= src)`?

Reply:
I'm concerned that you might run off the end of `buf` not that `dst` might get ahead of `src`. Nothing in this function ensures you can't keep going over the end of the buf AFAICT.

Reply:
Actually, this is all moot since as with the strcmp heuristic to pick a log level this sort of manipulation of the console stream has no place in Hyperkit so there is no point debating the implementation.